examples/cae/cae-slurm.yaml

```diff
@@ -0,0 +1,254 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+#
+#           ****************
+#######  CAE Solution Blueprint #######
+#           ****************
+#
+# This blueprint features a reference design suited for CAE applications on GCP.
+# It sets up the following infrastructure:
+#   * Google's H3 VMs, ideally suited for CAE workloads
+#   * Google's C3-highmem VM, suited for workloads with 16GB/core requirement
+#   * Google's Filestore NFS-based shared storage
+#   * Google's Chrome Remote Desktop
+#   * SLURM workload scheduler
+#
+blueprint_name: cae-slurm
+vars:
+  project_id:
```

Comment:
nit: usually we add ` ## Set GCP Project ID Here ##` to the project_id, to make it clear for the user that they have to add their project id.

Reply:
Yes, I agree. This makes things clearer. Resolved.